doc/modules.rst

```diff
@@ -30,20 +30,37 @@ Registration
 A module must import ``mig/modules``.
 
 A module registers itself at runtime via its ``init()`` function which must
-call ``modules.Register`` with a module name and a "runner function" returning
-an anonymous object (``interface {}``) that must implement the
+call ``modules.Register`` with a module name and an object implementing
```

Comment:
Actually, [go does have objects](https://golang.org/doc/faq#Is_Go_an_object-oriented_language).  It's not endowed with all of the behaviors a Java or C++ programmer might expect -- just like a C object -- but the term is still valid